internal/runners/push/push.go

```diff
@@ -197,31 +241,39 @@ func (r *Push) Run(params PushParams) error {
 	return nil
 }
 
-func (r *Push) getNamespace() (*project.Namespaced, error) {
+func (r *Push) verifyInput() error {
```

Comment:
nit: this isn't really verifying any input as auth is a state and the project is sourced from the project file. That said I don't immediately have a better alternative name.

Reply:
Yeah it's a little bit of a stretch, but close enough. I'd still call the user being authenticated or not "input".